tidb-cloud/migrate-data-into-tidb.md

```diff
@@ -87,16 +87,16 @@ You can use several ways to dump data from MySQL, such as using `mysqldump` or `
 
 ## Step 3. Import data to TiDB Cloud
 
-Depending on the location and size of your source data, the importing methods are different.  
+Depending on the location and size of your source data, the importing methods are different.
 
 - If your source data is located in Amazon S3 cloud storage, take the following steps:
 
-    1. Configure Amazon S3 access to allow TiDB cloud to access the source data in your Amazon S3 bucket. For more information, see [configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access). 
-    2. From the TiDB Cloud console, navigate to the TiDB Clusters page, and then click the name of your target cluster to go to its own overview page. In the upper-right corner, click **Import Data**, and then fill in the importing related information on the **Data Import Task** page.
+    1. Configure Amazon S3 access to allow TiDB cloud to access the source data in your Amazon S3 bucket. For more information, see [configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access).
+    2. From the TiDB Cloud console, navigate to the **Clusters** page, and then click the name of your target cluster to go to its own overview page. In the upper-right corner, click **...**, select **Import Data**, and then fill in the importing related information on the **Data Import** page.
```

Comment:
In the current environment, after clicking the cluster name to go to the cluster details page, "import data" is not listed in ...